pkg/controllers/certificaterequest_controller.go

```diff
@@ -1,207 +1,207 @@
-/*

```

Comment:
Why does this whole file show as changed? Can you please see if you can fix that so we preserve some Git history?

Reply:
Will look into it